packages/office-ui-fabric-react/src/components/List/examples/List.Grid.Example.tsx

```diff
@@ -75,65 +64,60 @@ const classNames: IListGridExampleClassObject = mergeStyleSets({
   },
 });
 
-const ROWS_PER_PAGE = 3;
-const MAX_ROW_HEIGHT = 250;
-
-export class ListGridExample extends React.Component<IListGridExampleProps> {
-  private _columnCount: number;
-  private _columnWidth: number;
-  private _rowHeight: number;
-  private _items: IExampleItem[];
-
-  constructor(props: IListGridExampleProps) {
-    super(props);
-    this._items = props.items || createListItems(5000);
-  }
+export const ListGridExample: React.FunctionComponent = () => {
+  let columnCount: number = 0;
```

Comment:
I'm still not sure this is right. It does appear to work, but it feels wrong and I'm not convinced the values would always be persisted as expected. Storing the values in `React.useRef()` would ensure persistence (and is the closest analogue to the old code) but still feels wrong. @dzearing do you have any thoughts on the actual right/idiomatic thing to do here?

(Also, you can get rid of columnWidth and just set rowHeight directly instead.)

Reply:
yeah move columnwidth to a local variable within the function it's used in; it's not read anywhere.

rowheight is read, and well, we could store the value in something like a state object. We need to cache it for the lifetime of the component, but setting it should not trigger re-renders. Something like useRef(0), and we cache the value in `current`.